gradle.properties

```diff
@@ -15,4 +15,6 @@
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# tl;dr: Projects are coupled if they change each others config and ours are not
+org.gradle.parallel=true
```

Comment:
reading through the docs linked in the comments of the `gradle.properties`:

> It may not be immediately apparent, but using key Gradle features like the allprojects and subprojects keywords automatically cause your projects to be coupled. This is because these keywords are used in a build.gradle file, which defines a project. Often this is a “root project” that does nothing more than define common configuration, but as far as Gradle is concerned this root project is still a fully-fledged project, and by using allprojects that project is effectively coupled to all other projects. Coupling of the root project to subprojects does not impact 'configuration on demand', but using the allprojects and subprojects in any subproject's build.gradle file will have an impact.

since we do use `allprojects` and `subprojects` in our android's `build.gradle` we cannot enable this flag

https://docs.gradle.org/current/userguide/multi_project_builds.html#sec:decoupled_projects